client/lib/signup/step-actions.js

```diff
@@ -102,6 +103,12 @@ export function createSiteOrDomain( callback, dependencies, data, reduxStore ) {
 	}
 }
 
+// We are experimenting making site topic a separate step from the survey.
+// Once we've decided to fully move away from the survey form, we can just keep the site topic here.
+function getSiteVertical( state ) {
+	return getSignupStepsSiteTopic( state ) || getSurveyVertical( state ).trim();
```

Comment:
I know it's trimmed at the input, but I'm wondering if there is any risk by not trimming it again before sending to the endpoint?

Reply:
I always honor defensive programming principles so why not. Trimming the return value as a whole in 0532848.